### DISCUSSION TITLE
Does sharing on r2modman save config?

### DISCUSSION THREAD

**Discussion by theSourPatchKing** [qwbarch/mirage#106] [Q&A] [answered]
If I share my mod profile with my friends, does it share it with the config options set to what I have it as? Im using r2modman.

**Comment by theSourPatchKing** [comment]
Another question: do I need to have masked overhaul to see suit colors on masked, and if so, will changing the spawn config to vanilla settings in mirage change masked overhauls spawn setting to vanilla as well?

**Comment by qwbarch** [comment, marked as answer]
R2modman shares the config, yes. Even if it doesn't, Mirage syncs the config so that every player is using the host's settings.
Regarding your second question, you don't need MaskedEnemyOverhaul to see suit colours. Mirage supports this out of the box
